account_move_exception/views/account_view.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="action_account_test_tree" model="ir.actions.act_window">
+        <field name="name">Account Exception Rules</field>
+        <field name="res_model">exception.rule</field>
+        <field name="view_mode">tree,form</field>
+        <field name="view_id" ref="base_exception.view_exception_rule_tree" />
+        <field
+            name="domain"
+        >[('model', 'in', ['account.move', 'account.move.line'])]</field>
+        <field
+            name="context"
+        >{'active_test': False, 'default_model' : 'account.move'}</field>
+    </record>
+    <menuitem
+        id="menu_account_exception_parent"
+        parent="account.menu_finance_configuration"
+        sequence="20"
```

Comment:
Can you add the group exception manager here? Other wise I get an error when accessing and trying to modify an exception: 

```
This operation is allowed for the following groups:
	- Exception manager
```

If I am not exception manager I should not have access

Reply:
Internal user have read access so there will no error

Reply:
But it is ugly to throw a exception like this one:

![account_move_exception](https://user-images.githubusercontent.com/19620251/182365654-b4b7d7ec-3bd1-4410-a481-79fdb94398f7.gif)

I think it makes sense to restrict the group

Reply:
Please try after hard refresh, you won't get create/edit button

![image](https://user-images.githubusercontent.com/94696387/182366223-f46a31bd-c04b-4245-bbdc-992a49a0982c.png)

![image](https://user-images.githubusercontent.com/94696387/182366264-b0540320-bbeb-4f77-8936-9016e427e0e9.png)

Reply:
I reproduced that in runboat, all the changes are available there I think. When do you restricted that?

In any case I think this is a bad decision when creating the module in previous versions and the group must be assigned to the menu items, only the mangers should see and edit, IMHO. Indeed the base_exception module restricts the security group.

Reply:
@AaronHForgeFlow I have added a group for the menu

Thanks